coniferest/session/__init__.py

```diff
@@ -93,17 +93,19 @@ def __init__(self, data, metadata, decision_callback = prompt_decision_callback,
 
         self._decision_cb = decision_callback
 
+        if on_refit_callbacks is None:
+            on_refit_callbacks = []
         if not isinstance(on_refit_callbacks, list):
```

Comment:
Should we use `elif` here?

Reply:
I believe it is a code-style question, I try to avoid `else` / `elif` when possible